Gemfile

```diff
@@ -25,5 +25,11 @@ else
   gem 'actionpack', gem_version
 end
 
+group :test do
+  gem 'activerecord'
+  gem 'sqlite3', platform: :ruby
+  gem 'activerecord-jdbcsqlite3-adapter', platform: :jruby
+end
```

Comment:
you're probably right that all the rails stuff above should be in a `gem :test` group..